lib/logstash/plugin_mixins/jdbc.rb

```diff
@@ -205,26 +207,29 @@ def execute_statement(statement, parameters)
       query = @database[statement, parameters]
       sql_last_value = @use_column_value ? @sql_last_value : Time.now.utc
       @tracking_column_warning_sent = false
-      @logger.debug? and @logger.debug("Executing JDBC query", :statement => statement, :parameters => parameters, :count => query.count)
+      @logger.debug? and @logger.debug("Executing JDBC query", :statement => statement, :parameters => parameters)
 
-      if @jdbc_paging_enabled
-        query.each_page(@jdbc_page_size) do |paged_dataset|
-          paged_dataset.each do |row|
+      @database.transaction do
```

Comment:
Thanks for describing this change in the PR. We really need this documented here as well.

Do you know how this will play with other databases? I'm not anticipating problems ATM, but I'm curious what your thoughts are. I'm wondering if this codepath should be PG only.